lib/trainer/options.rb

```diff
@@ -33,7 +33,13 @@ def self.available_options
                                      env_name: "TRAINER_FAIL_BUILD",
                                      description: "Should this step stop the build if the tests fail? Set this to false if you're handling this with a test reporter",
                                      is_string: false,
-                                     default_value: true)
+                                     default_value: true),
+        FastlaneCore::ConfigItem.new(key: :xcpretty_naming,
+                   short_option: "-x",
+                   env_name: "TRAINER_XCPRETTY_NAMING",
+                                     description: "Produces class name and test name identical to xcpretty naming in junit file",
+                   is_string: false,
+                                     default_value: false)
```

Comment:
This is a bit nit picky but can we align this config item arguments up with the ones above? 🙃